pkg/codegen/docs/gen_kubernetes.go

```diff
@@ -39,6 +39,11 @@ func (mod *modContext) isKubernetesOverlayModule() bool {
 		strings.HasPrefix(mod.mod, "helm") || strings.HasPrefix(mod.mod, "yaml")
 }
 
+func (mod *modContext) isComponentResource() bool {
+	// TODO: Support this more generally. For now, only the Helm and YAML overlays use ComponentResources.
```

Comment:
Isn't the [`apiextensions/CustomResource`](https://www.pulumi.com/docs/reference/pkg/kubernetes/apiextensions/customresource/) also a `ComponentResource`? Could we use `isKubernetesOverlayModule()` which is also a method on `modContext` (defined in this same file).

Reply:
No, it's a `CustomResource`

```
export class CustomResource extends pulumi.CustomResource
```

Reply:
Ah ok. 👍